test/wallet/index.ts

```diff
@@ -199,9 +217,278 @@ describe('Wallet', function () {
       const wallet = new Wallet(publicKey, privateKey)
       const signedTx: string = wallet.signTransaction(txJSON)
 
-      // TODO: Check the output of the signature against a known result
```

Comment:
Why was this TODO removed without being resolved?

Reply:
This test is probably not needed because it's the same thing as the one right under it